zpay32/invoice.go

```diff
@@ -87,7 +87,10 @@ const (
 var (
 	// InvoiceFeatures holds the set of all known feature bits that are
 	// exposed as BOLT 11 features.
-	InvoiceFeatures = map[lnwire.FeatureBit]string{}
+	InvoiceFeatures = map[lnwire.FeatureBit]string{
+		lnwire.TLVOnionPayloadRequired: "tlv-onion",
+		lnwire.TLVOnionPayloadOptional: "tlv-onion",
+	}
```

Comment:
`tlv-onion` is implictly signaling mpp support right?

Reply:
Ok, found out that `PaymentAddress` is the actual signal

Reply:
for now, yes. i think there's still some spec discussion as to whether we want a feature bit. i think we will end up adding one so that the receiver can require the parser to understand payment addresses, and fail during parsing rather than just getting a payment failure

Reply:
removed `InvoiceFeatures` entirely since flat features was merged